tools/nl/eval/README.md

```diff
@@ -9,3 +9,41 @@ python3 fetch_explore_queries.py
 
 This produces a csv file that can be imported to Google Sheet for further manual
 curation to produce golden data for eval purpose.
+
+## Prepare Model and Embedding Endpoints
+
+To evaluate a model and embeddings generated from the model, it needs to be
+deployed to Vertex AI. Concrete steps can be found in
+[model server](../../../model_server/README.md).
+
+Once a model is deployed, generate embeddings and index them in Vertex AI vector
+search. Concreate steps can be found in [build
+embeddings](../embeddings/README.md).
+
+Once model and embeddings endpoints are avaiable, record the endpoints id in
+`vertex_ai_endpoints.yaml` file.
+
+## Eval Unit
+
+One evaluation unit locates in a sub folder that contains a `golden.json` file
```

Comment:
nit: "one evaluation unit locates in a sub folder that contains a golden.json file" confuses me, what do you mean by this? Is each golden.json file an evaluation unit?

Reply:
Re-work this paragraph. A unit is simply a setup with one golden.json file. This golden file could be different for base DC or UN given the same query strings.